Makefile

```diff
@@ -230,7 +230,7 @@ $(EXECUTABLE): $(SOURCES)
 	$(GO) build $(GOFLAGS) $(EXTRA_GOFLAGS) -tags '$(TAGS)' -ldflags '-s -w $(LDFLAGS)' -o $@
 
 .PHONY: release
-release: release-dirs release-windows release-linux release-darwin release-copy release-check
+release: release-dirs release-windows release-linux release-darwin release-copy release-check release-compress
```

Comment:
Should `compress` come before `check`? I think in `check` the sha256 sums are calculated.

Reply:
Yes I think of it but it will generate a hash of the compressed version and I wasn't thinking it was usefull.

Reply:
Oh I think of one. For example in AUR package the hash is checked when we download so if the package is base on compressed version it will need it.

Reply:
Change done